app/models/condition.rb

```diff
@@ -270,6 +270,9 @@ def self.import_from_hash(condition, options = {})
     msg_pfx = "Importing Condition: guid=[#{condition["guid"]}] description=[#{condition["description"]}]"
 
     if c.nil?
+      c = Condition.find_by(:name => condition["name"])
+      c = Condition.find_by(:description => condition["description"]) if c.nil?
+      c&.destroy
```

Comment:
not sure if it would work, but can we use an `or` to lookup multiple records at the same time?

If all 3 fields need to be unique, then it seems any of these matches need to be resolved.

Reply:
👍